cypress/integration/HomeworkTests/calculator.spec.js

```diff
@@ -0,0 +1,155 @@
+/// <reference types="cypress" />
+
+describe('Asych', function () {
+    beforeEach(function () {
+        cy.visit('https://testsheepnz.github.io/BasicCalculator')
+
+    })
+
+    it('dropdown', function (){
+        cy.get('#selectBuild').select('1')
+        cy.get('#number1Field').type('3')
+        cy.get('#number2Field').type('3')
+        cy.get('#selectOperationDropdown').select('Multiply')
+        cy.get('#calculateButton').click()
+        cy.get('#numberAnswerField').should('have.value', '9')
+
+    })
+})
+
+
+describe('Asych', function () {
```

Comment:
Ar tikrai šita funkcija reikalinga? Pabandyk be jos (pasinaudok cypress dokumentacija iš ankstesnių komentarų, manau tikrai pagelbės)